### ISSUE TITLE
landscape pages may fail WCAG 2.1 (1.3.4)

### ISSUE THREAD

**Issue by teepeemm** [latex3/tagging-project#1356]
The following:
```tex
\DocumentMetadata{
  tagging=on, lang=en-US, pdfstandard=ua-2,
  tagging-setup={math/setup=mathml-SE} 
}
\documentclass{article}
\usepackage{pdflscape}
\title{Title}
\begin{document}
before
\begin{landscape}
landscape
\end{landscape}
after
\end{document}
```
is valid according to the default veraPDF.  But according to veraPDF's WCAG-2-2.xml profile, it fails WCAG2.2's clause 1.3.4.  The test in veraPDF requires that "Pages shall have the same orientation".

Clause 1.3.4 in WCAG 2.2 (and 2.1) is a level AA and says "Content does not restrict its view and operation to a single display orientation, such as portrait or landscape, unless a specific display orientation is essential."

I would think that a PDF already restricts itself to a single display orientation.  On the other hand, changing the orientation would definitely mess with how a reader has oriented their device.  I would think that the vast majority of PDF use cases would not have an "essential" display orientation, but I don't see how veraPDF could determine that.

So I suppose we could argue this is a spurious error on veraPDF's part, because a user could have an essential reason.  Or we could have LaTeX trigger some sort of warning that a user would have to disable if they promised they had an essential reason for a landscape page.

**Comment by cfr42** [comment]
this seems like a potentially counterproductive and somewhat arbitrary requirement for PDF?

**Comment by u-fischer** [comment]
Going from `Devices can be used in any orientation.`  (which doesn't make much sense for PDF) to `Pages shall have the same orientation` is a rather surprising logical jump. I would report that to veraPDF. @bdoubrov ?

**Comment by bdoubrov** [comment]
See my other comment in a different issue: https://github.com/latex3/tagging-project/discussions/1357#discussioncomment-17086142 .

I can just repeat that veraPDF is not doing WCAG validation. And yes, I do agree that this would be a human check anyway, as there might indeed be examples where different page orientation in PDF is fully legitimate and does not break accessibility.

**Comment by u-fischer** [comment]
@bdoubrov  sorry but I do not understand the logic. The wcag criterion about orientation says "Don't lock content to either portrait or landscape presentation." If you apply that to a PDF no PDF should pass that test as every PDF looks the orientation. WCAG doesn't say anything at all about different orientations at all so I do not understand why it should (only) fail  in a PDF if there is a rotate somewhere.

**Comment by bdoubrov** [comment]
The idea behind this check was that documents with mixed page orientation might be difficult to perceive by people with low vision or cognitive problems. 

I don't think there is an exact analog of this case in WCAG. The reference to WCAG 1.3.4 SC was chosen as a related one.

**Comment by u-fischer** [comment]
@bdoubrov sorry but I don't think that it is good idea to invent your own rule and then claim that it is based on a WCAG rule that targets something different only because the word "orientation" appears in both rule. That only confuses people.

**Comment by bdoubrov** [comment]
Totally agree and accept the critics. We'll fix this in the scope of PDF4WCAG

**Comment by FrankMittelbach** [comment]
@bdoubrov just a quick check: has this been adjusted or is in the pipeline and we can close here?

**Comment by bdoubrov** [comment]
This check was removed from WCAG 2.2 Machine profile in the latest release PDF4WCAG 1.10, and from the WCAG 2.2 Human (Machine & Human) profile in the latest dev version 1.11.*.
